distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex.enso

```diff
@@ -149,7 +155,8 @@ type Regex
         Helpers.expect_text input <|
            self.match_all input . map match_to_group_maybe
 
-    ## Splits the `input` text based on the pattern described by `self`.
+    ## GROUP Conversions
+       Splits the `input` text based on the pattern described by `self`.
```

Comment:
I'm a bit surprised to find this in `Conversions`.

It does make some sense (it is parsing the text into parts, and stuff like parse-to-integer _are_ conversions), but it is not really changing the type (it's still text, just in parts).

Reply:
It felt correct to have it with parse (which is a conversion) so split and tokenize ended up here.

Reply:
I see, ok.